www_src/pages/discover/discover.jsx

```diff
@@ -1,42 +1,43 @@
-var React = require('react');
+var React = require('react/addons');
+var ImageLoader = require('react-imageloader');
+
+var api = require('../../lib/api.js');
 var render = require('../../lib/render.jsx');
-var Binding = require('../../lib/binding.jsx');
 var Link = require('../../components/link/link.jsx');
-var api = require('../../lib/api.js');
 
 var Discover = React.createClass({
-  mixins: [Binding],
+  mixins: [],
   getInitialState: function () {
+    this.list = require('./mock.json');
+
     return {
-      test: false
+      list: this.list
     };
   },
-  componentWillMount: function () {
-    api({
-      uri: '/c0645e6953e9949f8e5c/raw/'
-    }, function (err, body) {
-      console.dir(err);
-      console.dir(body);
-    });
-  },
-  onClick: function () {
-    this.setState({test: !this.state.test});
-  },
   render: function () {
-    return (
-      <div id="discover" className="demo">
-        <Link url="/map/123" href="/pages/map" className="tile tile-demo">
-          <div className="inner">
-            <div className="img">
-              <img src="../../img/toucan.svg" />
-            </div>
+    var cards = this.state.list.map( project => {
+      return (
+        <Link url={"/map/" + project.id} href="/pages/map" key={project.id} className="card">
+          <div className="thumbnail">
+            <ImageLoader src={project.thumbnail[480]}>
+              // @todo Show image error icon / graphic
+            </ImageLoader>
+          </div>
+
+          <div className="meta">
```

Comment:
might be worth turning this into a `<Metadata>` component. Right now we're using title and author only, but we probably want to extend that later on with additional things like date posted, likes, etc.